### PR TITLE
[WC-2202] Enabling Playwright workflow

### PR DESCRIPTION
**Pull request by leonardomendix**
### Pull request type

Test related change (New E2E test, test automation, etc.)

---

### Description

As the first step of this migration, we can migrate the test and enable the execution in the CI using the workflow for Firefox E2E testing. This workflow runs weekly and shouldn’t affect our E2E test for every PR. 

### What should be covered while testing?
Automation should pass.

### CODE REVIEW

File: automation/run-e2e/playwright.config.cjs [hunk 1]
```diff
@@ -0,0 +1,78 @@
...
+        // },
+    ]
+
+    /* Run your local dev server before starting the tests */
+    // webServer: {
+    //   command: 'npm run start',
+    //   url: 'http://127.0.0.1:3000',
+    //   reuseExistingServer: !process.env.CI,
+    // },
+});
```
Comment (iobuhov) [inline automation/run-e2e/playwright.config.cjs:78]: Should we cleanup it a bit?

File: automation/run-e2e/playwright.config.cjs [hunk 2]
```diff
@@ -0,0 +1,78 @@
+// @ts-check
```
Comment (iobuhov) [inline automation/run-e2e/playwright.config.cjs:1]: why do you need @ts-check?

File: .github/workflows/RunE2EPlaywright.yml [hunk 3]
```diff
@@ -0,0 +1,55 @@
...
+                  node-version-file: ".nvmrc"
+                  cache: "pnpm"
+            - name: Install dependencies
+              run: pnpm install
+            - name: Install Playwright Browsers
+              run: npx playwright install --with-deps chromium
+            - name: Run build and run test project
+              run: npm run e2e.playwright
+            - name: Run E2E tests
+              run: npx playwright test --project=${{ matrix.project }} --shard=${{ matrix.shardIndex }}/${{ matrix.shardTotal }}
```
Comment (iobuhov) [inline .github/workflows/RunE2EPlaywright.yml:48]: I think it's better if you replace `npx` with `pnpm dlx`

File: automation/run-e2e/playwright.config.cjs (hunk 1 above)
Comment (leonardomendix) [reply automation/run-e2e/playwright.config.cjs]: ✅  Done!

File: automation/run-e2e/playwright.config.cjs (hunk 2 above)
Comment (leonardomendix) [reply automation/run-e2e/playwright.config.cjs]: Oops, removed.

File: .github/workflows/RunE2EPlaywright.yml (hunk 3 above)
Comment (leonardomendix) [reply .github/workflows/RunE2EPlaywright.yml]: Good one, thanks!

File: .github/workflows/RunE2EPlaywright.yml [hunk 4]
```diff
@@ -0,0 +1,55 @@
...
+              uses: actions/setup-node@5e21ff4d9bc1a8cf6de233a3057d20ec6b3fb69d # v3.8.1
+              with:
+                  node-version-file: ".nvmrc"
+                  cache: "pnpm"
+            - name: Install dependencies
+              run: pnpm install
+            - name: Install Playwright Browsers
+              run: pnpm exec playwright install --with-deps firefox
+            - name: Run build and run test project
+              run: npm run e2e.playwright
```
Comment (iobuhov) [inline .github/workflows/RunE2EPlaywright.yml:46]: Please, `pnpm` it 😄

Comment (leonardomendix) [reply .github/workflows/RunE2EPlaywright.yml]: Oops, I did it again. 🤣 
Fixed! ✅

**Review by rahmanunver — APPROVED** [review]

**Comment by rahmanunver** [comment]
e2e tests failing though

**Comment by leonardomendix** [comment]
> e2e tests failing though

Thanks, I'm waiting for the pie chart fix PR to rebase this one.

**Review by gjulivan — APPROVED** [review]

**Review by gjulivan — APPROVED** [review]

**Review by rahmanunver — APPROVED** [review]

**Pull request merged**